RealmSwift/ObjectiveCSupport+AnyRealmValue.swift

```diff
@@ -20,6 +20,21 @@
 import Foundation
 import Realm
 
+/// :nodoc:
+extension RLMDictionary: RLMValue {
+    /// :nodoc:
+    public var rlm_anyValueType: RLMAnyValueType { .dictionary }
```

Comment:
This is part of the obj-c API and it doesn't make any sense to be redefining it here.